### PR TITLE
转换规则 No. 362

### PR DESCRIPTION
**Pull request by co63oc**
### PR Docs

https://github.com/PaddlePaddle/PaConvert/issues/112

映射文档 https://github.com/PaddlePaddle/docs/pull/6114

362 torch.special.softmax

### PR APIs

```bash

```

### CODE REVIEW

File: paconvert/api_mapping.json [hunk 1]
```diff
@@ -10262,6 +10262,19 @@
       "out"
     ]
   },
+  "torch.special.softmax": {
+    "Matcher": "FunctionalSoftmaxMatcher",
```
Comment (zhwesky2010) [inline paconvert/api_mapping.json:10266]: 使用这个FunctionalSoftmaxMatcher的原因是`torch.nn.functional.softmax/log_softmax` 与 Paddle存在某些差异，torch的dim可以不输入，且默认计算的值与paddle不同。

infoflow 2023-08-23 20-14-46

`torch.special.softmax` 的dim是必须输入的，所以没这个问题，这里可以直接使用`GenericMatcher`即可

Comment (co63oc) [reply paconvert/api_mapping.json]: 已修改

**Review by zhwesky2010 — APPROVED** [review]
LGTM
